swarmrl/checkpointers/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""
+Package to hold all SwarmRL Checkpointers.
+"""
+
+from swarmrl.checkpointers.backup_checkpointer import BackupCheckpointer
+from swarmrl.checkpointers.best_checkpointer import BestRewardCheckpointer
+from swarmrl.checkpointers.checkpointer import Checkpointer
```

Comment:
This will seem pedantic but can we change `Checkpointer` to `BaseCheckpointer` to be very clear that it is a parent class.